Revision detection from git doesn't work with Debian packaging tags

Revision detetion from git done in Makefile doesn't work with tags created by Debian packaging tools (for example tag: debian/1.3.0-1linz_trusty1).

`make` will finish with following error:
```
$ make
sed -e 's/@@VERSION@@/1.3.0/' META.json.in > META.json
sed -e 's/@@VERSION@@/1.3.0/;s/@@REVISION@@/debian/1.3.0-1linz_trusty1/' sql/20-version.sql.in > sql/20-version.sql
sed: -e expression #1, char 45: unknown option to `s'
Makefile:66: recipe for target 'sql/20-version.sql' failed
make: *** [sql/20-version.sql] Error 1
make: *** Deleting file 'sql/20-version.sql'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -54,7 +54,7 @@ before_install:
 
 script:
   # Test https://github.com/linz/postgresql-tableversion/issues/47
-  - git tag "travis/1.2.3-4-gdeadbeef"
+  - git tag -a "travis/1.2.3-4-gdeadbeef" -m "test"
   - make
   - sudo env "PATH=$PATH" make install
   - PGPORT=5432 make installcheck || { cat regression.diffs; false; }

--- a/.travis.yml
+++ b/.travis.yml
@@ -53,6 +53,8 @@ before_install:
   - sudo apt-get install debhelper fakeroot postgresql-server-dev-all
 
 script:
+  # Test https://github.com/linz/postgresql-tableversion/issues/47
+  - git tag "travis/1.2.3-4-gdeadbeef"
   - make
   - sudo env "PATH=$PATH" make install
   - PGPORT=5432 make installcheck || { cat regression.diffs; false; }

--- a/Makefile
+++ b/Makefile
@@ -63,7 +63,7 @@ $(EXTENSION)--$(EXTVERSION).sql: sql/$(EXTENSION).sql
 	cp $< $@
 
 %.sql: %.sql.in
-	$(SED) -e 's/@@VERSION@@/$(EXTVERSION)/;s/@@REVISION@@/$(REVISION)/' $< > $@
+	$(SED) -e 's/@@VERSION@@/$(EXTVERSION)/;s|@@REVISION@@|$(REVISION)|' $< > $@
 	
 $(META): $(META).in
 	$(SED) -e 's/@@VERSION@@/$(EXTVERSION)/' $< > $@

